src/test/firecracker/micro-http-parsed-request/ignore-main.rs

```diff
@@ -5,26 +5,26 @@
 
 // Should return a nondet string of up to n characters
 // Currently RMC does not support strings
-fn rmc::nondet_string(n: u32) -> String {
+fn rmc::any_string(n: u32) -> String {
```

Comment:
Should this belong in the prelude?

Reply:
We don't currently support generating strings. We should consider doing something similar to what @zhassan-aws did for slices. But I think this is out of the scope of this PR.